proof/courier.go

```diff
@@ -1053,8 +1053,11 @@ func (c *UniverseRpcCourier) DeliverProof(ctx context.Context,
 		proofAsset := transitionProof.Asset
 
 		// Construct asset leaf.
+		// TODO(george): We pass in a 0 dec display as we have no easy
```

Comment:
Actually, the whole serialized asset doesn't really matter in the proof courier case. So IMO you can remove the TODO and just add a comment that the courier currently doesn't look at the RPC marshaled asset anyway.